shenyu-examples/shenyu-examples-springcloud/pom.xml

```diff
@@ -91,6 +82,32 @@
     </build>
 
     <profiles>
+        <profile>
+            <id>nacos</id>
+            <dependencies>
+                <dependency>
+                    <groupId>com.alibaba.cloud</groupId>
+                    <artifactId>spring-cloud-starter-alibaba-nacos-discovery</artifactId>
+                    <version>${nacos-discovery.version}</version>
+                </dependency>
+            </dependencies>
+            <activation>
+                <activeByDefault>false</activeByDefault>
+            </activation>
+        </profile>
+        <profile>
+            <id>eureka</id>
```

Comment:
Have you modified the .github/workflows/integrated-test.yml?
because you add two profiles

Reply:
No, I didnt noticed this yaml, thanks of remind .